crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -1183,14 +1256,16 @@ module deepbook::clob_v2 {
             );
         };
 
-        let order_id;
         if (restriction == IMMEDIATE_OR_CANCEL) {
             return (base_quantity_filled, quote_quantity_filled, false, 0)
         };
+
         if (restriction == FILL_OR_KILL) {
             assert!(base_quantity_filled == quantity, EOrderCannotBeFullyFilled);
             return (base_quantity_filled, quote_quantity_filled, false, 0)
         };
+
+        let order_id;
```

Comment:
Is there any point declaring this outside of the `if` -- it never escapes the respective branches.

Reply:
It was this way, so I decided to not intervene much. Makes sense to patch!